sql/src/test/java/io/crate/analyze/relations/RelationNormalizerTest.java

```diff
@@ -198,6 +199,42 @@ public void testSubSelectOnJoins() throws Exception {
     }
 
     @Test
+    public void testSubSelectOnOuterJoins() throws Exception {
+        QueriedRelation relation = normalize(
```

Comment:
using other test tables + column to make it more readable (and understandable ;) would be really great :)
(also at the other new tests here)